Unity-Technologies/ScriptableRenderPipeline
# CoreRP/Catch srp loading error while assetdatabase not available [Need C++ PR]

## Purpose of this PR
Remove error while SRP load resources at creation and the AssetDatabase is already importing
Need trunk PR: 
- https://ono.unity3d.com/unity/unity/pull-request/96486/_/graphics/catch-srp-loading-error-while-assetdatabase-not-available (2020.1)
- https://ono.unity3d.com/unity/unity/pull-request/96488/_/2019.3/graphics/catch-srp-loading-error-while-assetdatabase-not-available (2019.3)

Linked FogBugz: https://fogbugz.unity3d.com/f/cases/1188429/

---
### Testing status

**Manual Tests**: 
- Other: 
Manually tested that while HDRP package is being upgraded to newest version (7.1.1 to last HDRP/staging where this branch where before rebasing) remove every exception regarding the AssetDatabase not ready.
Also temporarily edited trunk code to verify that filtered exception where the ones that we wanted to remove.

**Automated Tests**: 

---
### Comments to reviewers
@phi-lira can you check that the last commit cover your issue and there is no side effect on universal? (And 2D)
